cfme/fixtures/nuage.py

```diff
@@ -5,10 +5,9 @@
 from cfme.utils.log import logger
 
 
-@pytest.fixture
-def with_nuage_sandbox(networks_provider):
+def create_nuage_sandbox(networks_provider):
```

Comment:
May I suggest to name it `create_basic_sandbox` instead? In the future we plan to come up with an extended sandbox as well (with Nuage templates and all) where we'll want to first create this basic setup and then extend it.

Reply:
Also, please modify the method to accept `nuage` as parameter, as suggested by the comment below.

Reply:
I modified both function name and input parameter as suggested.